### ISSUE TITLE
it doesn't work in Ubuntu 18.04,and I have no idea how to fix it

### ISSUE THREAD

**Issue by abcdef123ghi** [kylemanna/docker-openvpn#402]
my enviroment is not Ubuntu 18.04  and docker version is 18.5
I tried the following steps,however it didn't work
 $export OVPN_DATA=/home/myuser/workspace/OpenVPN_docker_community/openvpn_data

$docker run -v $OVPN_DATA:/etc/openvpn --log-driver=none --rm kylemanna/openvpn ovpn_genconfig -u udp://VPN.SERVER

$sudo docker run -v $OVPN_DATA:/etc/openvpn --log-driver=none --rm -it kylemanna/openvpn ovpn_initpki

$ sudo docker run -v $OVPN_DATA:/etc/openvpn --rm --name openvpn -p 1194:1194/udp --cap-add=NET_ADMIN kylemanna/openvpn
iptables: No chain/target/match by that name.
iptables: No chain/target/match by that name.
Enabling IPv6 Forwarding
sysctl: error setting key 'net.ipv6.conf.all.disable_ipv6': Read-only file system
Failed to enable IPv6 support
sysctl: error setting key 'net.ipv6.conf.default.forwarding': Read-only file system
Failed to enable IPv6 Forwarding default
sysctl: error setting key 'net.ipv6.conf.all.forwarding': Read-only file system
Failed to enable IPv6 Forwarding

and I tried 
$ sudo cat openvpn.log
Fri Jun 15 00:55:45 2018 OpenVPN 2.4.4 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Nov  9 2017
Fri Jun 15 00:55:45 2018 library versions: LibreSSL 2.6.3, LZO 2.10

I have no idea what wrong I have done,I tired to disabled ipv6 ,and afterward it didn't work as well

**Comment by mathieu-aubin** [comment]
in my opinion, docker isn't able to change the configuration file /etc/sysctl.conf 
That file usually is where such things as `'net.ipv6.conf.all.disable_ipv6` and other are located.

you need to make sure the file has the correct permissions to allow docker to edit it (i suppose -- i'm not docker litterate really)

When you decide to run docker with sudo, stick to it -- don't run one docker command with and the other without. Try and see if that does anything.

Also, make sure your configuration files are existing before you start the container (have a backup, just in case, who knows)

hope it helps

**Comment by kintesh** [comment]
Did you figure it out?

**Comment by victor23d** [comment]
It does work in Ubuntu 18.04 Bionic Beaver, I have tested on both ubuntu18 server and ubuntu18 client proved successful result. Just go through the basic docs, no additional configuration needed.
So it doesn't matter with Ubuntu18.

**Comment by abcdef123ghi** [comment]
thanks

**Comment by shaunstoltz** [comment]
Agreed, this dosnt work. Basic default install and it just dosnt work. I think this only works on a totally fresh install, maybe only fit for a dropplet or something similar, but this isnt ready for prime time at all.
